packages/tocco-ui/src/Select/Menu.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import TetherComponent from 'react-tether'
+import {components} from 'react-select'
+import _omit from 'lodash/omit'
+
+const StyledTether = styled(TetherComponent)`
+&& {
+  z-index: 10000000000000;
+`
+
+const StyledMenu = styled(components.Menu)`
+ && {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 14px;
+     margin-bottom: 0px;
+     margin-top: 3px;
+     width: ${props => props.wrapperWidth}px;
+     position: inherit;
+    .tether-target-attached-top & { 
+      transform: translateY(-${props => props.wrapperHeight + 6}px);
+    }
+ }
+`
+
+class Menu extends React.Component {
+  render() {
+    const props = this.props
+    return (
+      <StyledTether
+        attachment="top left"
+        targetAttachment="bottom left"
+        constraints={[{
+          to: 'window',
+          attachment: 'together'
+        }]}
+      >
+        <div></div>
```

Comment:
Yes. The first child of the Tether component is the one element that the tethered component (second child) gets attached to. Since we cannot wrap it around the select we must use an empty div.